test/ansible_processor_test.rb

```diff
@@ -16,9 +16,12 @@ def test_keys_are_strings
     define_method("test_snapshot_#{fixture_filename}") do
       processor = setup_processor("ansible", fixture_filename)
       result = processor.build_report
+      result2 = processor.build_facts
       remove_volatile_keys(result)
-      snapshot_filename = File.join(File.dirname(__FILE__), "snapshots", "#{File.basename(fixture_filename)}.snapshot.json")
+      snapshot_filename = File.join(File.dirname(__FILE__), "snapshots", "#{File.basename(fixture_filename)}.report.snapshot.json")
+      snapshot_filename2 = File.join(File.dirname(__FILE__), "snapshots", "#{File.basename(fixture_filename)}.facts.snapshot.json")
       assert_snapshot(result, snapshot_filename)
+      assert_snapshot(result2, snapshot_filename2)
```

Comment:
Please consider naming these as `result_report` and `result_facts`. Those "two" variables are bad :-)